documentation/blog/2024-02-01-whats-new-in-svelte-february-2024.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "What's new in Svelte: February 2024"
+description: 'New in Kit: `reroute`, `emulate` and more!'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Since SvelteKit 2.0 released late last year, there's been a bunch of new features to make the Kit dev experience even better! Meanwhile the Svelte team has been hard at work making Svelte 5 a reality.
+
+In case you missed it, Rich also posted [Tenets](https://github.com/sveltejs/svelte/discussions/10085#discussion-6029409), "an attempt to articulate the Svelte philosophy — our bedrock principles, that guide our design decisions."
+
+So let's dive in to check out what's new and see what the rest of the community has been up to...
+
+## What's new in SvelteKit
+
```

Comment:
maybe mention the new `import { read } from '$app/server'` feature here, even though it's mentioned Learning Resources as well?

I think some people might gloss over Learning Resources and they are just interested in new features in SvelteKit/Svelte

Reply:
good catch! I'm not sure how I missed that 😅